analyze/aps-bind.c

```diff
@@ -180,7 +180,7 @@ static SCOPE add_ext_sig(SCOPE old, Declaration tdecl, Signature sig) {
 	      current_type_env = saved;
 	      return new_scope; }
 	  default:
-	    /*! eventually handle renamings */
+	    /*! eventually handle remaining */
```

Comment:
Bad change.

renaming is what I want.  We are not remaining, but renaming things.